build.sbt

```diff
@@ -81,16 +77,15 @@ lazy val commonJsSettings = Seq(
   requiresDOM := false,
   jsEnv := new org.scalajs.jsenv.nodejs.NodeJSEnv(),
   // batch mode decreases the amount of memory needed to compile scala.js code
-  scalaJSOptimizerOptions := scalaJSOptimizerOptions.value.withBatchMode(isTravisBuild.value),
-  doctestGenTests := Seq.empty,
-  doctestWithDependencies := false
+  wptions.value.withBatchMode(isTravisBuild.value),
```

Comment:
looks like github's merge tool got this wrong. this line should be 
```scala
  scalaJSOptimizerOptions := scalaJSOptimizerOptions.value.withBatchMode(isTravisBuild.value),
```
@fthomas can you update it?

Reply:
I've rebased the commit onto master.